iiasa/message_ix
# Changes in the reporting related to mapping of addon technologies

This PR adds a mapping to the reporting init.py file to relate technologies with their addons in reporting.

## How to review

This PR is required to be able to review [PR#105](https://github.com/iiasa/message_data/pull/105) in message_data, related to the reporting of electricity generation.

- pull the changes in this PR in your message_ix repo
- pull the changes in [PR#105](https://github.com/iiasa/message_data/pull/105) into your message_data repo
- run the reporting script (for more info please see [PR#51](https://github.com/iiasa/message_data/pull/51))

- [x] New test is not needed (should pass the existing reporting test).
- [x] Documentation not needed.
